src/autoscaler/scalingengine/db/scalingengine.db.changelog.yml

```diff
@@ -38,25 +38,25 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: reason
-                  type: varchar
+                  type: varchar(255)
```

Comment:
we probably will need larger size for reason, message and error

Reply:
@boyang9527 Thank you for your comment. How about changing the data type to a longer type such as TEXT?

![screenshot from 2017-09-20 09-29-09](https://user-images.githubusercontent.com/16567880/30621457-2ea196b8-9de6-11e7-85c4-a617d5ddb3b5.png)